server/pages/index.jade

```diff
@@ -110,25 +110,45 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
-		if user
+		if supportUser
+			script.
+				window.supportUser = {
+					readyScripts: []
+				};
+		if user && !supportUser
 			script(type='text/javascript')!='var currentUser = ' + sanitize.jsonStringifyForHtml( user )
 		if app
 			script(type='text/javascript')!='var app = ' + sanitize.jsonStringifyForHtml( app )
-		if initialReduxState
+		if initialReduxState && !supportUser
```

Comment:
Server rendering and subsequent passing of the data used for that render to the client in `initialReduxState` will only ever happen when no user is logged in, and there are no plans to do this for logged-in users, so I think this discussion is academic.

If we did theoretically want to make sure to not use original user data at all, we'd have to do so at an earlier point, to avoid the server rendering with that data. For now I would simply remove this check, since `initialReduxState` would never be populated when supportUser is active (assuming supportUser only affects logged-in operation).